src/nvme/ioctl.c

```diff
@@ -438,6 +438,8 @@ int nvme_get_log_page(int fd, __u32 xfer_len, struct nvme_get_log_args *args)
 	void *ptr = args->log;
 	int ret;
 
+	args->fd = fd;
```

Comment:
That's the only line I change in the first patch... Hmm, you mean unrelated to the series? Without setting the fd here the new user in nvme-cli doesn't work, because the fd is not set in the args struct. Let me extend the commit message to make it more clear why it's needed.

Reply:
Ah! Sorry, I was reviewing through the files-changed UI. All good if this is in its own change.

Reply:
No worries. I got fooled by the UI several times, thus I started to look at the changes using the github cli tool (https://cli.github.com):

```shell
$ gh pr list
$ gh pr checkout $NUMBER
$ tig
```

Works way better for me :)